app/backend/src/couchers/urls.py

```diff
@@ -30,10 +30,14 @@ def password_reset_link(password_reset_token):
     return f"{config['BASE_URL']}/password-reset/{password_reset_token}"
 
 
-def host_request_link():
+def host_request_link_host():
     return f"{config['BASE_URL']}/messages/hosting/"
 
 
+def host_request_link_guest():
+    return f"{config['BASE_URL']}/messages/surfing/"
+
+
```

Comment:
Update //docs/urls.md with these urls. Later on we want to have the conversation id in there too I guess.